src/symbolize/gimli/coff.rs

```diff
@@ -54,15 +54,15 @@ impl<'a> Object<'a> {
         let mut i = 0;
         let len = symtab.len();
         while i < len {
-            let sym = symtab.symbol(i).ok()?;
+            let sym = symtab.symbol(SymbolIndex(i)).ok()?;
```

Comment:
It would be better to use `symtab.iter()`, instead of reimplementing the aux symbol handling here.

Reply:
Done!